manifests/timer.pp

```diff
@@ -0,0 +1,107 @@
+# Create a timer and optionally a service unit
```

Comment:
this should be `# @summary Create a timer and optionally a service unit to execute with the timer unit`?